src/encoded/schemas/library.json

```diff
@@ -67,6 +67,33 @@
                 }
             ]
         },
+        "nucleic_acid_term_name": {
+            "oneOf": [
+                {
+                    "required": ["strand_specificity"],
+                        "properties": {
```

Comment:
I think the indentation here is off?

Reply:
properties should be same indentation as required

Reply:
Ah you're right- the section below has an extra indent also (L87-92), fixed both